manifest.cfg

```diff
@@ -0,0 +1,19 @@
+[default]
+path =
+include-files = changelog.txt,LICENSE.md
+include-directories =
+
+[runtime]
+path = runtime
+exclude-files = lua-profiler.lua,msvcr100.dll,SimpleGraphic.cfg,Update.exe
+exclude-directories =
+
+[src]
```

Comment:
Old part name was "program", this changes it to "src", possibly messing with the update script (haven't verified yet, but would be good to change it back for the first run at least to verify no changes)